Pipfile

```diff
@@ -84,6 +82,8 @@ pylint = "*"
 pylint_celery = "*"
 isort = "*"
 yapf = "*"
+django-debug-toolbar = "==1.11" # The Request History plugin isn't compatible with 2.0
```

Comment:
We could just tell pipenv to install `<2`. So we don't block any `1.x` patch (even if it is very unlikely to happen).